analyses/optimus/process_10x_v2_datasets.ipynb

```diff
@@ -0,0 +1,652 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "import shutil\n",
+    "import json\n",
+    "import os\n",
+    "from google.cloud import storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## create submission packages for each of the new datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "### Make Directories"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "testdir = os.environ['TMPDIR'] + 'process_10x_v2_datasets'\n",
+    "os.environ['testdir'] = testdir\n",
+    "os.makedirs(testdir, exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## Make the dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "# grab the dependencies; these are static\n",
+    "workflow_dependencies = {\n",
+    "    \"StarAlignBamSingleEnd.wdl\": \"../../../skylab/library/tasks/StarAlignBamSingleEnd.wdl\",\n",
```

Comment:
Are all these paths with ../../../ always going to work?

Reply:
No. I'll make a variable for the skylab directory. Thanks for the comment.